test/integration/targets/cnos_vlan/meta/main.yaml

```diff
@@ -0,0 +1,2 @@
+dependencies:
+#  - prepare_ios_tests
```

Comment:
If you don't have a platform setup playbook, you don't need this file

Reply:
Removing this now